packages/faustwp-cli/src/index.ts

```diff
@@ -75,10 +75,9 @@ import { marshallTelemetryData, sendTelemetryData } from './telemetry/index.js';
   if (getWpSecret()) {
     try {
       const telemetryData = marshallTelemetryData(arg1);
-      void sendTelemetryData(telemetryData);
+      await sendTelemetryData(telemetryData);
```

Comment:
Without this, the http call wasn't reliably making it to WP before the `process.exit()` call below.

Reply:
Is there a way to use an eslint rule to capture that check?

https://typescript-eslint.io/rules/promise-function-async/

https://github.com/typescript-eslint/typescript-eslint/issues/4553